Remotion/Web/Development.WebTesting/Utilities/AssertionExceptionUtility.cs

```diff
@@ -28,62 +29,70 @@ public static class AssertionExceptionUtility
   {

```

Comment:
I believe my remarks regarding making the driver parameter not-nullable (and then only adding an overload in the backport got lost.